recipe/symfony.php

```diff
@@ -107,26 +120,152 @@
     run('{{env_vars}} {{bin/php}} {{bin/console}} cache:warmup {{console_options}}');
 })->desc('Warm up cache');
 
+/**
+ * Create database if not exists
+ */
+task('database:create', function () {
+    if (env('enable_database_create')) {
+        run('{{env_vars}} {{bin/php}} {{bin/console}} doctrine:database:create --if-not-exists {{console_options}}');
+    }
+})->desc('Create database if not exists (only if set `enable_database_create` true)');
 
 /**
  * Migrate database
  */
 task('database:migrate', function () {
-    run('{{env_vars}} {{bin/php}} {{bin/console}} doctrine:migrations:migrate {{console_options}}');
-})->desc('Migrate database');
+    if (env('use_database_migration_strategy')) {
+        run('{{env_vars}} {{bin/php}} {{bin/console}} doctrine:migrations:migrate {{console_options}}');
+    }
+})->desc('Migrate database (only if set `use_database_migration_strategy` true)');
+
+/**
+ * Database migration rollback
+ */
+task('database:migrate:rollback', function () {
```

Comment:
This is a really complicated task. There is no way to do it simple? 
This code reley on a lot of stuff what can be changed, and can bring a lot of bugs. 
But i like a work. Maybe it's better to create separate project for this task where it's possible to write tests for this?